pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1754,6 +1754,21 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes[0].Message).To(Equal(fmt.Sprintf("Invalid IOThreadsPolicy (%s)", ioThreadPolicy)))
 		})
 
+		It("should reject GPU devices when feature gate is disabled", func() {
+			vmi := v1.NewMinimalVMI("testvm")
+			vmi.Spec.Domain.Devices.Gpus = []v1.Gpu{
+				v1.Gpu{
+					Name: "nvidia.com/gpu_name",
+				},
+			}
+
+			//disableFeatureGates()
```

Comment:
let's remove this in the next respin.

Reply:
Removed it. I have addressed all the comments. I think this PR looks good now. Let me know what else we need to get it merged.